### PR TITLE
feat: Create Incoming Call UI

### PR DESCRIPTION
**Pull request by alexandreferris**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [X] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

There was no UI for incoming call

### Solutions

Incoming Call UI was implemented.

### Dependencies (Optional)

This is only the FiRST PR, more will follow soon, this is just the UI, no logic no nothing, currently the only way to open this screen is to click to start a call in a conversation (this will be changed later on as the logic is implemented in the following PRs)

Needs releases with:

#### How to Test
- Open the App
- Open a conversation
- Click on start a call/camera button at the top
- Screen should open and you can check the design

### Attachments
incoming call ui

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/SpeakerButton.kt [hunk 1]
```diff
@@ -2,21 +2,35 @@ package com.wire.android.ui.calling.controlButtons
...
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import coil.compose.rememberAsyncImagePainter
 import com.wire.android.R
 import com.wire.android.ui.theme.wireDimensions
-import com.wire.android.util.getUriFromDrawable
 
 @Composable
 fun SpeakerButton() {
-    Image(
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/SpeakerButton.kt:17]: had to wrap it in a `IconButton` so the dimensions were the same as the other buttons

File: app/src/main/kotlin/com/wire/android/ui/calling/incoming/IncomingCallScreen.kt [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+import com.wire.android.ui.common.topappbar.WireCenterAlignedTopAppBar
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+
+@OptIn(ExperimentalMaterialApi::class, ExperimentalMaterial3Api::class)
+@Composable
+fun IncomingCallScreen() {
+    IncomingCallContent()
+}
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/calling/incoming/IncomingCallScreen.kt:42]: Currently, no state/no viewmodel this PR is only for the UI without any logic

File: app/src/main/kotlin/com/wire/android/ui/common/topappbar/WireCenterAlignedTopAppBar.kt [hunk 3]
```diff
@@ -9,13 +9,15 @@ import androidx.compose.material3.Surface
...
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.unit.Dp
 import com.wire.android.ui.theme.wireDimensions
 import com.wire.android.ui.theme.wireTypography
 
 @Composable
 fun WireCenterAlignedTopAppBar(
     title: String,
+    titleStyle: TextStyle = MaterialTheme.wireTypography.title01,
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/common/topappbar/WireCenterAlignedTopAppBar.kt:20]: Added this title style do we don't need to create another screen/component exactly the same but with a different style.

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt [hunk 4]
```diff
@@ -105,9 +105,14 @@ class ConversationViewModel @Inject constructor(
...
+//                    destination = NavigationItem.OngoingCall.getRouteWithArgs()
+//                )
+//            )
             navigationManager.navigate(
                 command = NavigationCommand(
-                    destination = NavigationItem.OngoingCall.getRouteWithArgs()
+                    destination = NavigationItem.IncomingCall.getRouteWithArgs()
                 )
             )
         }
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt:118]: Just for now, will be changed on later PRs

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-509/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3096148)(3096148)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3096148)(3096148)-dev-release.apk)

**Review by trOnk12 — APPROVED** [review]
lgtm 🥮

**Review by ohassine — APPROVED** [review]
👌👌

**Pull request merged**